styles/button__section.css

```diff
@@ -0,0 +1,63 @@
+@import "normalize.css";
+
+html {
+    box-sizing: border-box;
+}
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+.title {
+    background: #FFF;
```

Comment:
Tutaj to też jest default value więc możemy usunąć

Reply:
nie wiem czemu się tak z tym białym rozpędziłem, zmienione + import Google Fonts, bo zauważyłem, że gdzieś mi uciekło